introspection-engine/introspection-engine-tests/tests/tables/mysql.rs

```diff
@@ -219,7 +219,7 @@ async fn a_table_with_descending_unique(api: &TestApi) -> TestResult {
     Ok(())
 }
 
-#[test_connector(tags(Mysql), preview_features("fullTextIndex"))]
+#[test_connector(tags(Mysql), preview_features("fullTextIndex"), exclude(TiDB))]
```

Comment:
Not supported in TiDB? Does an issue exist?

Reply:
Yes, TiDB currently does not support the FULLTEXT index. Related issue: https://github.com/pingcap/tidb/issues/1793

More about MySQL compatibility of TiDB, you can found it in the document:   https://docs.pingcap.com/tidb/dev/mysql-compatibility#unsupported-features